src/modules/index.js

```diff
@@ -2,6 +2,7 @@
  * Internal dependencies
  */
 import * as blocks from './blocks';
+import './fs_archive_events';
```

Comment:
Quick and dirty way to include to primary compiler. I assume we will want to get it loaded via our `./blocks` above? Something else?